vnext/ReactWindowsCore/Modules/PlatformConstantsModule.cpp

```diff
@@ -28,6 +40,12 @@ std::map<std::string, folly::dynamic> PlatformConstantsModule::getConstants() {
       // better source of truth later. Tracked by Issue #4073
       {"reactNativeVersion", folly::dynamic::object("major", 0)("minor", 61)("patch", 0)}
 
+      // Provide version information for react-native-windows -- which is independant of
+      // the version of react-native we are built from
+      {"reactNativeWindowsVersion",
```

Comment:
Out of curiosity, are there places where we know we want to check this, or is this more just forward looking?

Reply:
We're going to want a similar check to what fb has, esp once we start shipping a dll and it would be easier to get them out of sync.